core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/views/SixteenByNineFrameLayout.kt

```diff
@@ -16,11 +17,51 @@ open class SixteenByNineFrameLayout: FrameLayout {
     constructor(context: Context, attrs: AttributeSet?) : this(context, attrs, 0)
     constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) : super(context, attrs, defStyleAttr)
 
+    var mCustomTouchListener: OnTouchListener? =null
+
     override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
         if (layoutParams.height == ViewGroup.LayoutParams.WRAP_CONTENT) {
             val sixteenNineHeight = View.MeasureSpec.makeMeasureSpec(View.MeasureSpec.getSize(widthMeasureSpec) * 9 / 16, View.MeasureSpec.EXACTLY)
             super.onMeasure(widthMeasureSpec, sixteenNineHeight)
         } else
             super.onMeasure(widthMeasureSpec, heightMeasureSpec)
     }
+
+    override fun onInterceptTouchEvent(ev: MotionEvent): Boolean {
+        /*
```

Comment:
The goal of this comment is to document the method, not implementation details. Instead of a comment inside the method, put kdoc on the method. Further reading [here](https://kotlinlang.org/docs/reference/kotlin-doc.html#kdoc-syntax).